### ISSUE TITLE
release-3.1.0: gene panel API seems to be returning null

### ISSUE THREAD

**Issue by adamabeshouse** [cBioPortal/cbioportal#6480]
e.g. https://beta.cbioportal.org/results/oncoprint?=&Action=Submit&RPPA_SCORE_THRESHOLD=2&Z_SCORE_THRESHOLD=1&cancer_study_list=coadread_tcga_pub&case_set_id=coadread_tcga_pub_nonhypermut&clinicallist=0%2C2%2CMETHYLATION_SUBTYPE&data_priority=0&gene_list=KRAS%20NRAS%20BRAF&geneset_list=%20&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=coadread_tcga_pub_gistic&genetic_profile_ids_PROFILE_MRNA_EXPRESSION=coadread_tcga_pub_rna_seq_mrna_median_Zscores&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=coadread_tcga_pub_mutations&heatmap_track_groups=coadread_tcga_pub_rna_seq_mrna_median_Zscores%2CKRAS%2CNRAS%2CBRAF&show_samples=false&tab_index=tab_visualize

**Comment by n1zea144** [comment]
Hi @adamabeshouse.  When I run this query against release-3.1.0, I see the following exception thrown on the backend:
`
ERROR org.cbioportal.web.util.InvolvedCancerStudyExtractorInterceptor - exception thrown during extraction of genePanelSampleMolecularIdentifiers: com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot deserialize instance of `org.cbioportal.web.parameter.SampleMolecularIdentifier[]` out of START_OBJECT token                                                                                                                                                                
 at [Source: (BufferedReader); line: 1, column: 1]
`

Here is a snippet of a request that comes into the backend that is causing this error
```json
{
  "sampleMolecularIdentifiers": [
    {
      "molecularProfileId": "coadread_tcga_pub_gistic",
      "sampleId": "TCGA-A6-2670-01"
    },
    {
      "molecularProfileId": "coadread_tcga_pub_rna_seq_mrna",
      "sampleId": "TCGA-A6-2670-01"
    },
    {
      "molecularProfileId": "coadread_tcga_pub_rna_seq_mrna_median_Zscores",
      "sampleId": "TCGA-A6-2670-01"
    },
... 
  ]
}
```

 **Is it possible that the construction of the request has changed on the front-end?**  I ask because the format described and the request constructed via the [swagger page](https://beta.cbioportal.org/api/swagger-ui.html#/O._Gene_Panels/fetchGenePanelDataInMultipleMolecularProfilesUsingPOST) is in a slightly different format than what is being sent by the results page (no 'sampleMolecularIdentifiers' map/wrapper):

```json
[
  {
    "molecularProfileId": "coadread_tcga_pub_gistic",
    "sampleId": "TCGA-A6-2670-01"
  },
  {
    "molecularProfileId": "coadread_tcga_pub_rna_seq_mrna",
    "sampleId": "TCGA-A6-2670-01"
  },
...
]
```

**Comment by adamabeshouse** [comment]
got it. that makes sense. I'll check that out, thanks

**Comment by adamabeshouse** [comment]
fixed now in frontend release-3.1.0
